src/Observers/RunwayObserver.php

```diff
@@ -41,13 +41,25 @@ public function updating(Model $model)
         return false;
     }
 
+    public function retrieved(Model $model)
+    {
+        $originalAttributes = $model->getAttributes();
+        
+        if ($model->exists && $modelEntry = ($model->entry ?? '')) {
+            $model->setRawAttributes(array_merge(
+                $modelEntry->data()->toArray(),
```

Comment:
Might be cleaner to just do `$model->entry->data()->toArray()` and skip setting a variable within the if statement.